src/components/utils/test/helper.cc

```diff
@@ -35,22 +35,25 @@ namespace test {
 namespace components {
 namespace utils_test {
 
+const int_least32_t kRetryPeriodMs = 100;
+
 MainAndWorkerThreadSynchronisation::MainAndWorkerThreadSynchronisation()
     : done_cond_()
     , done_lock_()
     , done_auto_lock_(done_lock_)
     , var_lock_()
     , job_is_done(false) {}
 
-void MainAndWorkerThreadSynchronisation::WaitJobDone(const uint32_t milliseconds) {
-  while (true) {
+void MainAndWorkerThreadSynchronisation::WaitJobDone(int_least32_t miliseconds) {
```

Comment:
@orhan-mehmedov int_least32_t  is interesting type, newer seen it before. Bt i prefere use there something like size_t